docs/reference/buildx_du.md

```diff
@@ -20,6 +20,70 @@ Disk usage
 
 ## Examples
 
+### Show disk usage
+
+The `docker buildx du` command shows the disk usage for the currently selected
+builder.
+
+```console
+$ docker buildx du
+ID                                RECLAIMABLE    SIZE          LAST ACCESSED
+12wgll9os87pazzft8lt0yztp*        true           1.704GB       13 days ago
+iupsv3it5ubh92aweb7c1wojc*        true           1.297GB       36 minutes ago
+ek4ve8h4obyv5kld6vicmtqyn         true           811.7MB       13 days ago
+isovrfnmkelzhtdx942w9vjcb*        true           811.7MB       13 days ago
+0jty7mjrndi1yo7xkv1baralh         true           810.5MB       13 days ago
+jyzkefmsysqiaakgwmjgxjpcz*        true           810.5MB       13 days ago
+z8w1y95jn93gvj92jtaj6uhwk         true           318MB         2 weeks ago
+rz2zgfcwlfxsxd7d41w2sz2tt         true           8.224kB*      43 hours ago
+n5bkzpewmk2eiu6hn9tzx18jd         true           8.224kB*      43 hours ago
+ao94g6vtbzdl6k5zgdmrmnwpt         true           8.224kB*      43 hours ago
+2pyjep7njm0wh39vcingxb97i         true           8.224kB*      43 hours ago
+Shared:        115.5MB
+Private:       10.25GB
+Reclaimable:   10.36GB
+Total:         10.36GB
+```
+
+If `RECLAIMABLE` is false, the `docker buildx du prune` command won't delete
+the record, even if you use `--all`. That's because the record is actively in
+use by some component of the builder.
+
+The asterisks (\*) in the output indicate the following:
+
+- An asterisk next to an ID (`zu7m6evdpebh5h8kfkpw9dlf2*`) indicates that the record
```

Comment:
We could provide some example of how this looks in the verbose record as well as it is much easier to understand that these are separate fields with their own values there, instead of knowing what `*` means.

Reply:
Yep, I can do a follow-up